joomla/joomla-cms
# Users Options - passwords

#### Summary of Changes

Cosmetic change to move the options for passwords to their own tab in com_users options and to change the strings - this should improve usability and remove confusion
#### Testing Instructions

Apply the PR and go got com_users options and see the new tab

After the PR it should look like the following

![jul4 1](https://cloud.githubusercontent.com/assets/1296369/16375848/705a4986-3c55-11e6-96d9-14e077b26bad.png)

![kpll](https://cloud.githubusercontent.com/assets/1296369/16375856/74e0d65a-3c55-11e6-9589-05e168e10d98.png)
